share/sysbuild/images/soc/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+include(${SOC_${SB_CONFIG_SOC}_DIR}/sysbuild.cmake OPTIONAL)
```

Comment:
this would need to be documented somewhere.

Perhaps as part of: https://github.com/zephyrproject-rtos/zephyr/pull/69475